recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+source:
+  url: https://github.com/conda-forge/conda-forge-build-setup-feedstock/raw/master/vs2008_patch.zip
```

Comment:
Also please see these, which are basically the same.

xref: https://github.com/conda-forge/staged-recipes/pull/251
xref: https://github.com/conda-forge/conda-smithy/pull/107